### PR TITLE
Navbar, Footer & Basic Folder Structure

### PR DESCRIPTION
**Pull request by MarkVed17**
- Created Placeholders for Screens
- Routing
- Added Navbar Component
- Added Footer Component
- Added Assets

Preview: 

recording

### CODE REVIEW

File: src/frontend/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./Navbar.css";
+
+function Navbar() {
+  return (
+    <>
+      <header className="header-nav nav-search">
+        <NavLink to="/" className="header-logo-link">
```
Comment (poornima-alugubelly) [inline src/frontend/components/Navbar/Navbar.js:9]: Didn't know that we have NavLink as well, nice!

File: src/frontend/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Footer.css";
+
+function Footer() {
+  return (
+    <>
+      <footer class="footer-container">
+        {/* Logo + Copyright */}
```
Comment (poornima-alugubelly) [inline src/frontend/components/Footer/Footer.js:9]: I like how u have appropriate comments

**Review by poornima-alugubelly — APPROVED** [review]

File: src/frontend/routes/Routes.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { Routes, Route } from "react-router-dom";
+import {
+  CartScreen,
+  HomeScreen,
+  ProductsScreen,
+  WishlistScreen,
+} from "../screens";
+import Mockman from "mockman-js";
+import { SignInScreen } from "../screens/SignInScreen/SignInScreen";
+import { SignUpScreen } from "../screens/SignUpScreen/SignUpScreen";
```
Comment (mimansha-swarup) [inline src/frontend/routes/Routes.js]: hey you can import it on line 3 with other screens 
this might have happened cause of auto import

**Review by mimansha-swarup — APPROVED** [review]

File: src/frontend/routes/Routes.js (hunk 3 above)
Comment (MarkVed17) [reply src/frontend/routes/Routes.js]: Thanks!
I'll do it.

File: src/frontend/screens/CartScreen/CartScreen.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function CartScreen() {
+  return (
+    <div
+      className="main-content"
+      style={{ color: "white", marginTop: "10rem", textAlign: "center" }}
```
Comment (t007rushi) [inline src/frontend/screens/CartScreen/CartScreen.js:7]: Use class name instead of inline styling

**Review by t007rushi — COMMENTED** [review]
One more suggestion to use compressed images as to enhance UX instead of png

File: src/frontend/screens/CartScreen/CartScreen.js (hunk 4 above)
Comment (MarkVed17) [reply src/frontend/screens/CartScreen/CartScreen.js:7]: You're right. This is just the placeholder styling. Will be anyways removed later.

**Pull request merged**